ui/org.eclipse.pde.ui/src/org/eclipse/pde/ui/launcher/JUnitTabGroup.java

```diff
@@ -48,9 +48,11 @@ public void setDefaults(ILaunchConfigurationWorkingCopy configuration) {
 		} catch (CoreException e) {
 			vmArgs = ""; //$NON-NLS-1$
 		}
-		vmArgs = AssertionVMArg.enableAssertInArgString(vmArgs);
-		if (vmArgs.length() > 0)
-			configuration.setAttribute(IJavaLaunchConfigurationConstants.ATTR_VM_ARGUMENTS, vmArgs);
+		@SuppressWarnings("restriction")
```

Comment:
This entire `src_samples` source folder somehow seems to be related to something like `sample.properties` files.
Does anybody know anything about that?

It also seems to be associated to a `org.eclipse.sdk.samples` provided via the site
http://www.eclipse.org/pde/samples/site.xml

But I have never heard of this and assume it's outdated if it is usable at all?

@merks, @iloveeclipse or @akurtakov do you know anything about that or can this entire source folder be removed?